pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -193,12 +195,14 @@ type PersistentVolumeClaimInfo interface {
 	GetPersistentVolumeClaimInfo(namespace string, name string) (*v1.PersistentVolumeClaim, error)
 }
 
+var _ PersistentVolumeClaimInfo = &CachedPersistentVolumeClaimInfo{}
```

Comment:
This seems unnecessary as these are just internal packages and the definition of the interface is in the same file.

Reply:
Yes, they are in the same file.
But I think it's still a good rule to always validate an interface implementation.
Such as:
- https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/controller_utils.go#L429-L443
- https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/cpumanager/cpu_manager.go#L52-L102

Reply:
> This seems unnecessary as these are just internal packages and the definition of the interface is in the same file.

I think it is a good pattern which could help us to find problems during compile time.

Reply:
Note that compilation of the scheduler would still fail, because we use only the interface outside of this package.

Reply:
@alculquicondor If each implementation follows this "compile-time assertion" pattern, potential issues can be found easily, and also it can speedup code refactoring if someday we want to change the interface.